custom_components/ha_creality_ws/light.py

```
@@ -11,37 +15,96 @@ async def async_setup_entry(hass, entry, async_add_entities):
+
+    @property
+    def brightness(self) -> int | None:
+        if not self._has_brightness_control:
+            return None
+        if not self._is_on():
+            return None
+        # This firmware never reports the dim level, so report the remembered
+        # value (or full brightness if we never set one).
+        return self._brightness if self._brightness is not None else 255
 
     async def async_turn_on(self, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `async_turn_on`

Add return type annotation: `None`

(ANN202)